### DISCUSSION TITLE
Setup-Error messages

### DISCUSSION THREAD

**Discussion by ghoethe** [jomjol/AI-on-the-edge-device#704] [Q&A]
At the End of the Setup I get following error messages:

In the selectd field the value '' in the parameter 'PostProcessing' in the field 'MaxRateType' is invalid.
PLEASE CHECK BEFORE SAVING!

In the selectd field the value '' in the parameter 'PostProcessing' in the field 'ExtendedResolution' is invalid.
PLEASE CHECK BEFORE SAVING!

In the selectd field the value '' in the parameter 'PostProcessing' in the field 'IgnoreLeadingNaN' is invalid.
PLEASE CHECK BEFORE SAVING!

<img width="661" alt="Bildschirmfoto 2022-02-24 um 17 19 43" src="https://user-images.githubusercontent.com/59373636/155564283-7360c707-a6d4-4298-8461-5b7556c09b4c.png">

**Comment by ghoethe** [comment]
i have had to activate all Postprocessing individual parameters, now the error message is gone

**Comment by jomjol** [comment]
Since the last version the content of the fields are testet for consistent context. This is done, independely if the option is activated or not. Maybe I change this in the next version.
